src/gui/editors/AutomationEditor.cpp

```diff
@@ -1412,7 +1432,17 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 	// draw current edit-mode-icon below the cursor
 	switch( m_editMode )
 	{
-		case DRAW: cursor = s_toolDraw; break;
+		case DRAW:
+			if( m_mouseDownRight )
+			{
+				cursor = s_toolErase;
+			}
+			else
```

Comment:
You could also add a change for the cursor when automation points are moved, like in the piano roll: https://github.com/LMMS/lmms/blob/85ed63be852d4da2555da49f93d8fd0e18cc3f6c/src/gui/editors/PianoRoll.cpp#L3136-L3152

So we keep consitency.